toolchain/sem_ir/stringify_type.cpp

```diff
@@ -26,17 +26,22 @@ static auto GetTypePrecedence(InstKind kind) -> int {
 }
 
 namespace {
+// Contains the stack of steps for `StringifyTypeExpr`.
 class StepStack {
  public:
+  // The discriminator for the `Step` union.
```

Comment:
This may be out of scope, but I feel like this would be clearer as a member of `Step`, so it can be documented together with `kind`.

Reply:
Done